src/alphabet.rs

```diff
@@ -123,6 +123,12 @@ impl Alphabet {
 
         Ok(Self::from_str_unchecked(alphabet))
     }
+
+    /// Create a String from the symbols in the `Alphabet`
+    #[cfg(any(feature = "std", test))]
+    pub fn as_string(&self) -> String {
+        self.symbols.iter().map(|c| *c as char).collect::<String>()
```

Comment:
This could avoid allocation, and thus be no_std compatible, by returning a &str: use std::str::from_utf8 on symbols. It's safe to unwrap/expect that, since symbols are verified to be printable ascii. Users in std environments can then .to_string() the str if needed.

Reply:
Ooh interesting! Would this look something like `std::str::from_utf8(self.symbols.as_ref()).unwrap()`

Reply:
If that's the case, I suppose the method should be more aptly named `as_str()`

Reply:
Yep, that should do it.

Reply:
Perfect. Pushed a new commit. Resolving!